recipes/diode/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "diode" %}
+{% set version = "1.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/diode-{{ version }}.tar.gz
+  sha256: bb863e036fa2c3a9ba19db1c462629abbaffbc731f20b524e9924f22c384b08e
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
+  entry_points:
```

Comment:
Empty entry points?

Reply:
yes

Reply:
Why?

Reply:
Why? This is very unusual. Please write a comment in the recipe to explain why it is needed. This recipe is hosted publicly and we get a lot of copypasta from people creating recipes from recipes without knowing how they work.

Reply:
Thanks for your feedback. Perhaps I don't know how they work. But I don't see any command line script or anything like that being defined by diode (https://github.com/mrzv/diode/blob/e77e9cea8f1be8e167dfcc8a6ee1770378df4859/setup.py#L59-L89).